packages/@ionic/cli/src/commands/package/build.ts

```diff
@@ -325,8 +346,8 @@ This can be used only together with build type ${input('release')} for Android a
     }
   }
 
-  async getDownloadUrl(appflowId: string, buildId: number, token: string): Promise<DownloadUrl> {
-    const { req } = await this.env.client.make('GET', `/apps/${appflowId}/packages/${buildId}/download`);
+  async getDownloadUrl(appflowId: string, buildId: number, artifactType: string, token: string): Promise<DownloadUrl> {
+    const { req } = await this.env.client.make('GET', `/apps/${appflowId}/packages/${buildId}/download?artifactType=${artifactType}`);
```

Comment:
I could've solved the 'default' artifact type by making it optional here and let the api handle it. The api did have comments in it to eventually make it required.

Reply:
👍🏻